tests/test_tasks/test_spatial.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, Tuple
+
+import pytest
+
+from sentinelhub import CRS, BBox
+
+from eogrow.tasks.spatial import get_array_slices
+
+
+@pytest.mark.parametrize(
+    "input_params, expected",
+    [
+        (
+            {
+                "bbox": BBox(((729480.0, 4390045.0), (732120.0, 4391255.0)), crs=CRS("32638")),
```

Comment:
Its hard to figure out wether the expected result really is what is expected, because BBoxes have such convoluted coordinates. Would be easier if they were simplified a bit, for instance make all points a bit above `(730000, 4400000)` so that we can focus on the changes in the lower digits without a calculator.

Reply:
good, now make some test cases less regular (make them wide 1110 meters instead of 1000 etc.)

Reply:
added